internal/validator/main.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"github.com/InjectiveLabs/sdk-go/client/common"
+	"github.com/InjectiveLabs/sdk-go/client/exchange"
+	injective_derivative_exchange_rpcpb "github.com/InjectiveLabs/sdk-go/exchange/derivative_exchange_rpc/pb"
+	injective_spot_exchange_rpcpb "github.com/InjectiveLabs/sdk-go/exchange/spot_exchange_rpc/pb"
+	"github.com/InjectiveLabs/utils-go/pkg/token"
+	log "github.com/sirupsen/logrus"
+	"os"
+	"strings"
+	"time"
+)
+
+var (
+	tokenFile *string
+)
+
+var (
+	// Invalid markets for tests that will be ignored
+	TestsMarkets = map[string]struct{}{
+		"0xed865fd44f1bc9d46d978db415ed00444fac4f6aef7e09e2d0235f8d140b219f": {},
+	}
+)
+
+func init() {
+	tokenFile = flag.String("token-file", "token.json", "path to token file")
+}
```

Comment:
nit: inits should be avoided if possible. In this case, we don't want to initialize anything special as we are not importing this package elsewhere. We might as well just move this to `main()`